app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadViewModel.kt

```diff
@@ -206,6 +207,15 @@ class ThreadViewModel @Inject constructor(
             }.getOrElse { ThreadHeaderVisibility.NONE }
         }
 
+    val isCollapsable: LiveData<Boolean> = MediatorLiveData<Boolean>()
+        .apply {
+            addSource(messagesLive) { (items, _) ->
+                val messageCount = runCatchingRealm { items.count { it is MessageUi } }.getOrDefault(0)
+                value = messageCount > 1
+            }
+        }
+        .distinctUntilChanged()
```

Comment:
The idea of keeping a LiveData instead of turning everything into a Flow is appreciable but here it's trivial to turn the messages into a flow and keep exposing a LiveData of messages nonetheless. This will help us get rid of live datas little by little